### PR TITLE
Add yarn so we can have some cool JS

### PR DESCRIPTION
**Pull request by Anbranin**
Didn't have yarn, could have yarn now. Need it on the server too I guess hopefully that's easy.

### CODE REVIEW

**Comment by Anbranin** [comment]
😮

File: Gemfile [hunk 1]
```diff
@@ -20,7 +20,6 @@ gem 'snappconfig'
 gem 'uglifier'
 
 source 'https://rails-assets.org' do
-  gem 'rails-assets-datetimepicker'
   gem 'rails-assets-moment'
```
Comment (werebus) [inline Gemfile]: Do you feel like tackling one more?

File: config/application.rb [hunk 2]
```diff
@@ -16,6 +16,9 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
     config.time_zone = 'Eastern Time (US & Canada)'
+    config.autoload_paths << Rails.root.join('lib')
```
Comment (werebus) [inline config/application.rb]: Why these? I think it's fine, but just curious if it matters for this PR in some way?

File: Gemfile (hunk 1 above)
Comment (Anbranin) [reply Gemfile]: I feel like trying to tackle one more, sure. I got pretty myopic to a point.

File: config/application.rb (hunk 2 above)
Comment (Anbranin) [reply config/application.rb]: I'm embarrassed to say I have no idea.

**Pull request merged**